<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
    <addaction name="action_Exit_All"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Edit"/>
    <addaction name="action_Delete"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_AboutProgram"/>
    <addaction name="action_AboutAuthor"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Add"/>
   <addaction name="action_Edit"/>
   <addaction name="action_Delete"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
  </widget>
  <action name="action_Language">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="action_AboutProgram">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="action_AboutAuthor">
   <property name="text">
    <string>About author</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/new_file.png</normaloff>:/icons/img/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/open_file.png</normaloff>:/icons/img/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/save_file.png</normaloff>:/icons/img/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit_All">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/add_property.png</normaloff>:/icons/img/add_property.png</iconset>
   </property>
   <property name="text">
    <string>Add record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/edit_property.png</normaloff>:/icons/img/edit_property.png</iconset>
   </property>
   <property name="text">
    <string>Edit record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/delete_property.png</normaloff>:/icons/img/delete_property.png</iconset>
   </property>
   <property name="text">
    <string>Delete record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/img/close_file.png</normaloff>:/icons/img/close_file.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
